umbraco/Umbraco.UI
# Docs/pimp my readmes

This PR adds a lot to the docs. It also adds plop as a dev dependency and a script that scaffold a new package.

How to test:
-read the docs and check if all is right
-run `npm run new-package` and then `npm i`
-go through the new package files and see if the name you picked is everywhere
-start sotybook and see if the story is rendered properly
-run test in watch mode and see if test is running
-trash the test package, do not commit
